### PR TITLE
Signed-off-by: jared.grant@ibm.com <jared.grant@ibm.com>

### PR DESCRIPTION
**Pull request by jgrant22**
add filler and order systems to Identifiers and change procedure Identifiers

### CODE REVIEW

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java [hunk 1]
```diff
@@ -726,161 +724,50 @@ public void procedureIdentifierTest() {
...
-        assertThat(value).isEqualTo("PON001"); // OBR.2.1
-        assertThat(system).isNull();
-        type = identifier.getType();
-        coding = type.getCoding().get(0);
-        assertThat(type.getText()).isNull();
-        assertThat(coding.getSystem()).isEqualTo("http://terminology.hl7.org/CodeSystem/v2-0203");
-        assertThat(coding.getCode()).isEqualTo("PLAC");
-        assertThat(coding.getDisplay()).isEqualTo("Placer Identifier");
-
-        // Test: ExtID from MSH-7, filler and placer from ORC, visit number from MSH-7
```
Comment (klwhaley) [inline src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:814]: I think we should have a test with PR1, but no PR1.19, so we have to fall back to MSH-7 for the identifier.

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java [hunk 2]
```diff
@@ -921,13 +808,13 @@ public void procedureIdentifierTest() {
...
-//          documentReference =
+//           documentReference =
 //                  "MSH|^~\\&|HL7Soup|Instance1|MCM|Instance2|200911021022|Security|MDM^T02^MDM_T02|64322|P|2.6|123|456|ER|AL|USA|ASCII|en|2.6|56789^NID^UID|MCM||||\n" +
-//                          "PID|1||000054321^^^MRN|||||||||||||M|CAT|||||N\n" +
-//                          "TXA|1||B45678|||||||||||PON001\n" +
-//                          "ORC|NW|||PGN001|SC|D|1||20170825010500|MS|MS||||20170825010500|\n" +
-//                          "OBR|1||CD_000000|2244^General Order|||20170825010500||||||Relevant Clinical Information|||||||002|||||F|||550600^Tsadok550600^Janetary~660600^Merrit660600^Darren^F~770600^Das770600^Surjya^P~880600^Winter880600^Oscar^||||770600&Das770600&Surjya&P^^^6N^1234^A|\n";
-//          report = ResourceUtils.getDocumentReference(documentReference);
+//                  "PID|1||000054321^^^MRN|||||||||||||M|CAT|||||N\n" +
+//                  "TXA|1||B45678|||||||||||PON001^IE|k|\n" +
```
Comment (klwhaley) [inline src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:814]: total nit, but did we add the 'k' on purpose? i don't think i see us use it in tests?

Comment (klwhaley) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:814]: OH i see, i think you are using it be sure you are getting TXA.14 and not TXA.15

**Review by klwhaley — CHANGES_REQUESTED** [review]
Please add the requested test, and fix the 3 failing tests, then I'll re-review! THanks!

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java (hunk 1 above)
Comment (jgrant22) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:814]: Ahh thanks !

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java (hunk 2 above)
Comment (jgrant22) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:814]: I honestly don't recall why I didn't just leave this blank

File: src/main/resources/hl7/resource/Procedure.yml [hunk 3]
```diff
@@ -10,14 +10,12 @@ id:
...
+   valueOf: datatype/Identifier_var
    generateList: true
    expressionType: resource
    vars:
-      system: SYSTEM_URL, $sys
-      value: OBR.44.1 | OBR.45.1 | MSH.7
-   constants:
-      sys: "urn:id:extID"
+      valueIn: PR1.19.1 | MSH.7
+      systemCX: PR1.19.2
```
Comment (klwhaley) [inline src/main/resources/hl7/resource/Procedure.yml:18]: Missed this in first review, Lisa and I were discussing.  Please update back to urn:id :extID
19.2 is not a system, it has to do with the number of char/digits of the code.

So leave value PR1.19.1 | MSH.7
But change system back how you had it.

Comment (jgrant22) [reply src/main/resources/hl7/resource/Procedure.yml]: thanks, fixed

**Review by klwhaley — APPROVED** [review]
Thank you for all the updates! Looks good, Please merge!

**Pull request merged**